jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/webdriver/WebDriverDelegatingInterceptor.java

```diff
@@ -18,16 +19,22 @@
 public class WebDriverDelegatingInterceptor extends IntroductionInfoSupport
     implements IntroductionInterceptor {
 
-    private WebDriverFactory webDriverFactory;
 
-    private WebDriver driver = null;
+    private transient WebDriverFactoryResolver webDriverFactoryResolver;
 
-    public WebDriverDelegatingInterceptor(WebDriverFactory webDriverFactory) {
-        this.webDriverFactory = webDriverFactory;
+    private transient WebDriver driver = null;
```

Comment:
Why transient?

Reply:
IntroductionInfoSupport implements Serializable (not 100% sure why to be honest) - so all members of subclasses must implement Serializable as well or be transient...